src/resources/audio.js

```diff
@@ -30,6 +30,22 @@ pc.extend(pc, function () {
     };
 
     AudioHandler.prototype = {
+        _isSupported: function (url) {

```

Comment:
We can pre-cache this in variable, instead of calling it every time on load event.